lib/build/amodro-trace/write/stubs.js

```diff
@@ -12,7 +13,9 @@ module.exports = function stubs(options) {
   options = options || {};
 
   return function(context, moduleName, filePath, contents) {
-    if (options.stubModules && options.stubModules.indexOf(moduleName) !== -1) {
+    if (options.stubModules
```

Comment:
Do we have a PR for Amodro-Trace to update to this? We've updated this library multiple times to support features we need, I'm wondering if we should just officially fork this to do our own loading, or we should continue to open PR's for the changes we make? @JeroenVinke, do you have an opinion?